### DISCUSSION TITLE
UNC Path

### DISCUSSION THREAD

**Discussion by evenzp** [casualsnek/cassowary#129] [Q&A] [answered]
Once I map my C: drive in the Windows -> Linux section, and it gives me a UNC path, then I mount it how do I access? Do I use a certain type of protocol for it? I've tried everything I know to access it... but that's not saying a lot.

I'm a bit new 😅

**Comment by casualsnek** [comment, marked as answer]
> Once I map my C: drive in the Windows -> Linux section, and it gives me a UNC path, then I mount it how do I access? Do I use a certain type of protocol for it? I've tried everything I know to access it... but that's not saying a lot.
> 
> I'm a bit new sweat_smile

Depends on your file manager / Desktop environment ! On dolphin/kde, it should be instantly visible on your devices section. In any case if mount was successful, you can find your mounted C drive at : ```/mnt/casualrdh/<drive_letter>``` or ```/mnt/casualrdh/c```

**Comment by evenzp** [comment]
Ahh I see, should've checked there. Didn't show on devices for me
